components/brave_news/browser/combined_feed_parsing.cc

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
```

Comment:
It would be a shame to lose the git history link to the previous file. Is it possible to split in to two commits - 1 that moves the file, and one that modifies?

Reply:
I have submitted it as a separate PR. I will rebase this one on top once that one is merged. https://github.com/brave/brave-core/pull/17424